iati/home/models.py

```diff
@@ -72,6 +72,7 @@ class IATIStreamBlock(StreamBlock):
     pullquote = PullQuoteBlock()
     aligned_html = AlignedHTMLBlock(icon="code", label='Raw HTML')
     document_box = DocumentBoxBlock(icon="doc-full-inverse")
+    skip_anchor = CharBlock(icon="order-down")
```

Comment:
I'm not sure about the term `skip_anchor`. At present, the name implies that you want to move further forward in a page. This might not be the case if it is intended to move back up to the page - e.g. 'Back to top'.

How about `anchor_point`?

Reply:
Also flagging that this functionality may become part of wagtail in future: https://github.com/wagtail/wagtail/issues/1049